fastkml/kml.py

```diff
@@ -997,6 +997,77 @@ def ImagePyramid(self, tileSize, maxWidth, maxHeight, gridOrigin):
         self.maxHeight = maxHeight
         self.gridOrigin = gridOrigin
 
+    def etree_element(self):
+        element = super().etree_element()
+        if self._rotation:
+            rotation = etree.SubElement(element, f"{self.ns}rotation")
+            rotation.text = self._rotation
+        if all([self._leftFov, self._rightFov, self._bottomFov, self._topFov, self._near]):
+            ViewVolume = etree.SubElement(element, f"{self.ns}ViewVolume")
+            leftFov = etree.SubElement(ViewVolume, f"{self.ns}leftFov")
+            leftFov.text = self._leftFov
+            rightFov = etree.SubElement(ViewVolume, f"{self.ns}rightFov")
+            rightFov.text = self._rightFov
+            bottomFov = etree.SubElement(ViewVolume, f"{self.ns}bottomFov")
+            bottomFov.text = self._bottomFov
+            topFov = etree.SubElement(ViewVolume, f"{self.ns}topFov")
+            topFov.text = self._topFov
+            near = etree.SubElement(ViewVolume, f"{self.ns}near")
+            near.text = self._near
+        if all([self._tileSize, self._maxWidth, self._maxHeight, self._gridOrigin]):
+            ImagePyramid = etree.SubElement(element, f"{self.ns}ImagePyramid")
+            tileSize = etree.SubElement(ImagePyramid, f"{self.ns}tileSize")
+            tileSize.text = self._tileSize
+            maxWidth = etree.SubElement(ImagePyramid, f"{self.ns}maxWidth")
+            maxWidth.text = self._maxWidth
+            maxHeight = etree.SubElement(ImagePyramid, f"{self.ns}maxHeight")
+            maxHeight.text = self._maxHeight
+            gridOrigin = etree.SubElement(ImagePyramid, f"{self.ns}gridOrigin")
+            gridOrigin.text = self._gridOrigin
+
+    def from_element(self, element):
+        super().from_element(element)
+        rotation = element.find(f"{self.ns}rotation")
+        if rotation is not None:
+            self.rotation = rotation.text
+        ViewVolume = element.find(f"{self.ns}ViewVolume")
+        if ViewVolume is not None:
+            leftFov = ViewVolume.find(f"{self.ns}leftFov")
+            if leftFov is not None:
+                self.leftFov = leftFov.text
+            rightFov = ViewVolume.find(f"{self.ns}rightFov")
+            if rightFov is not None:
+                self.rightFov = rightFov.text
+            bottomFov = ViewVolume.find(f"{self.ns}bottomFov")
+            if bottomFov is not None:
+                self.bottomFov = bottomFov.text
+            topFov = ViewVolume.find(f"{self.ns}topFov")
+            if topFov is not None:
+                self.topFov = topFov.text
+            near = ViewVolume.find(f"{self.ns}near")
+            if near is not None:
+                self.near = near.text
```

Comment:
consider adopting the shorter
```
if attribute:
```
syntax throughout the project?

Reply:
If it works, I remember that Lxmls elements had a falsy value - but this is 10 years in the past